test/util.test.ts

```diff
@@ -125,10 +125,6 @@ describe("util", () => {
   })
 
   describe("getOptions", () => {
-    // Things to mock
-    // logger
-    // location
-    // document
```

Comment:
Comment disappeared! no other changes in this file; curious about the change

Reply:
That was intentional! See commit: https://github.com/cdr/code-server/pull/2701/commits/164d11e0277c8c39171c0b5398461a67bdb3ac72